content/en/real_user_monitoring/session_replay/mobile/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Mobile Session Replay
+kind: documentation
+description: Describes how to setup Session Replay for mobile devices
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+<div class="alert alert-warning">
+Mobile Session Replay is in public beta for native mobile apps. As such, there is no billing for this feature at the time.
+</div>
+
+## Overview
+
+Mobile Session Replay expands visibility into your mobile applications by visually replaying each user interaction, such as taps, swipes, and scrolls. It is available for native apps on both Android and iOS. Simulating user interactions on your applications makes it easier to reproduce crashes and errors, as well as understand the user journey for making UI improvements.
```

Comment:
Why mention simulating here? Maybe I'm misunderstanding, but the point of RUM and session replay is that it's not simulated?

Reply:
True, will change this back to "Visually replaying"